File: php/Clothes.php
<?php
require_once "Accessories.php";

class Clothes extends Accessories
{
    private $category;
    private $size;
    private $brand;

    public function __construct($id, $name, $price, $stock, $image, $type, $material, $color, $category, $size, $brand)
    {
        parent::__construct($id, $name, $price, $stock, $image, $type, $material, $color);
        $this->category = $category;
        $this->size = $size;
        $this->brand = $brand;
    }

    // getter dan setter untuk category
    public function setCategory($category)
    {
        $this->category = $category;
    }

    public function getCategory()
    {
        return $this->category;
    }

    // getter dan setter untuk size
    public function setSize($size)
    {
        $this->size = $size;
    }

    public function getSize()
    {
        return $this->size;
    }

    // getter dan setter untuk brand
    public function setBrand($brand)
    {
        $this->brand = $brand;
    }

    public function getBrand()
    {
        return $this->brand;
    }
}

File: java/Clothes.java
public class Clothes extends Accessories {
    private String category;
    private String size;
    private String brand;

    // constructor dengan parameter
    Clothes(int id, String name, int price, int stock, String type, String material, String color, String category, String size, String brand) {
        super(id, name, price, stock, type, material, color);
        this.category = category;
        this.size = size;
        this.brand = brand;
    }

    // getter dan setter untuk category
    public void setCategory(String category) {
        this.category = category;
    }

    public String getCategory() {
        return category;
    }

    // getter dan setter untuk size
    public void setSize(String size) {
        this.size = size;
    }

    public String getSize() {
        return size;
    }

    // getter dan setter untuk brand
    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getBrand() {
        return brand;
    }
}
